api/v1alpha1/lua_types.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+// LuaFilter defines a Lua extension that applies globally
```

Comment:
The comment here could be misleading, as the EEP can apply at either the Gateway or HTTPRoute level.